src/ripple/protocol/LedgerFormats.h

```diff
@@ -243,6 +243,9 @@ enum LedgerSpecificFlags {
         0x10000000,               // True, reject new paychans
     lsfDisallowIncomingTrustline =
         0x20000000,               // True, reject new trustlines (only if no issued assets)
+    lsfAMM [[maybe_unused]] = 0x40000000, // True, AMM account
```

Comment:
Why are we adding a flag pertaining to `AMM` in this PR? I couldn't find any usages of this flag in the codebase, what is it used for?

Reply:
This flag is added in the AMM PR https://github.com/XRPLF/rippled/pull/4294. I added it as a placeholder to give more clarity for upcoming PRs or resolving conflicts.